### DISCUSSION TITLE
The pressure fluctuation

### DISCUSSION THREAD

**Discussion by Fahimeh-Goodarzi** [xcompact3d/Incompact3d#297] [Q&A] [answered]
Dear all,
When we save the pressure, we see that the pressure fluctuates from one time step to the next. Why is this so and how can we correct it? In some cases, we could fix the pressure at each time step by subtracting the pressure at a reference point where the pressure should remain constant. I don't believe we have such a point in our simulation. Another thought was to use the mean pressure on the outlet plane as our reference. Would it be more appropriate to use the mean pressure on the outlet plane as a reference?

**Comment by slaizet** [comment, marked as answer]
Hi,
Without proper details about your computational setup, it is going to be difficult to help you. I can just say that the pressure is defined with a constant, so using a reference point close to the boundaries (where nothing is happening) could be a good idea.
Best,
Sylvain

**Reply by Fahimeh-Goodarzi** [reply]
Thank you.
